src/app/core/components/add-marios-form/components/employee-select/employee-select.component.html

```diff
@@ -0,0 +1,34 @@
+<ng-select
+  class="search-field-multiselect employee-avatar custom-ng-select"
+  [items]="employees$ | async"
+  bindLabel="fullName"
+  [class.error-border]="control.invalid && control.touched"
+  placeholder="Who you want to recognize?"
+  appendTo="body"
+  [multiple]="true"
+  [(ngModel)]="selected"
+  [clearable]="false"
+  [formControl]="control"
+  (search)="onSearchChange($event)"
+  [searchFn]="false"
+  [required]="true"
+>
+  <ng-template ng-label-tmp let-item="item" let-clear="clear">
+    <span>{{ item.firstName + " " + item.lastName }}</span>
+    <span class="ng-value-icon left" (click)="clear(item)" aria-hidden="true"
+      >×</span
```

Comment:
It could be in line 18

Reply:
435d636a6c9597f805b50b9c664b0da498eed625